Software/Development/SemesterProject/App/referenceHandler.c

```diff
@@ -47,7 +47,7 @@
  */
 static GUISignalsTag GUISignals;
 static float speedReference, torqueReference;
-static float deltaSpeed = 0.05, deltaTorque = 0.005; //This values are set for execution at 1 kHz, original values were 1 and 0.1
+static float deltaSpeed = 0.05, deltaTorque = 1; //This values are set for execution at 1 kHz, original values were 1 and 0.1
```

Comment:
Be sure that this ramp of the torque reference is behaving as expected

Reply:
Isn't 1 a huge change in deltaTorque? why is this?

Reply:
Yes, maybe we should decrease it

Reply:
Agreed